core/server/data/schema/schema.js

```diff
@@ -438,6 +438,17 @@ module.exports = {
         },
         created_at: {type: 'dateTime', nullable: false}
     },
+    members_product_events: {
+        id: {type: 'string', maxlength: 24, nullable: false, primary: true},
+        member_id: {type: 'string', maxlength: 24, nullable: false, references: 'members.id', cascadeDelete: true},
+        product_id: {type: 'string', maxlength: 24, nullable: false, references: 'products.id', cascadeDelete: false},
+        action: {
+            type: 'string', maxlength: 50, nullable: true, validations: {
+                isIn: [['add', 'remove']]
```

Comment:
On the [actions table](https://github.com/TryGhost/Ghost/blob/50354633487558e4e4520d10a173182b979018cf/core/server/data/schema/schema.js#L548) the actions are past tense afaict - these should be consistent